test/issues/issue-80/unformatted.pug

```diff
@@ -0,0 +1,6 @@
+span.aui-lozenge(
+  :class=`{
+    "aui-lozenge-inprogress": pipeline.status == PipelineStatus.Running,
+    "aui-lozenge-error":      pipeline.status == PipelineStatus.Failed
+  }`
+)
```

Comment:
Could you add two more snippets: one with `"` (double quote) and one with `'` (single quote)
These two should be formatted as usual

I also found that adding an additional snippet with at least one indentation is always a good thing

e.g. [test/options/closingBracketPosition/last-line/unformatted.pug#L9 +](https://github.com/prettier/plugin-pug/blob/master/test/options/closingBracketPosition/last-line/unformatted.pug#L9)

Reply:
@Shinigami92 Thanks for the quick feedback. Sure, will do.

Reply:
@Shinigami92 done!